lib/possible_bipartition.rb

```diff
@@ -1,4 +1,39 @@
-
+#using bfs
 def possible_bipartition(dislikes)
-  raise NotImplementedError, "possible_bipartition isn't implemented yet"
+  if dislikes == []
+    return true
+  end
+  compat = {}
+  dislikes.each_with_index do |neighbors, index|
+    bfs(dislikes, compat, index)
```

Comment:
You are actually not doing anything with the return value for `bfs`.  So... maybe:

```suggestion
    unless bfs(dislikes, compat, index)
      return false
    end
```